### DISCUSSION TITLE
캐릭터 이동과 카메라 회전 관련

### DISCUSSION THREAD

**Discussion by hina9** [naverz/zepeto-studio-kor#729] [5. Q&A]
안녕하세요.

제페토로 캐릭터 움직임까지 튜토리얼을 보고 따라 작성하였는데,

예시의 조이스틱은 고정되있는 반면 새로 제작된 조이스틱은 범위가 있어 다른곳으로 계속 움직이더라구요..

이부분을 고정할 수 있는 방법은 따로 없나요?

그리구 카메라 회전 부분도 다른 제페토 월드의 움직임과는 확연하게 차이가 많이 나서  카메라 설정하는 방법이 따로 있는지 같이 문의드립니다.

https://user-images.githubusercontent.com/93582930/167586144-4a2d3817-01ad-4d2c-9726-2d9eb47bfde3.mp4

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 예전에 지원했던 v-pad prefab은 조이스틱이 고정형이었으나 1.1.6버전 이후로 현재의 프리팹으로 변경되었습니다. https://github.com/naverz/zepeto-studio-kor/discussions/602
만약 고정형을 원하신다면 월드 버전을 1.1.6 이전으로 변경하신 후 
프리팹을 복사해서 사용하는 방법이 있습니다.

또는 v-pad를 아예 끄고 NewInputSystem으로 제어하는 방법이 있습니다.
최근에 추가된 캐릭터 컨트롤 커스터마이즈 가이드 및 샘플을 참고해 주시면 도움이 될 것 같습니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/826

그리고 ZepetoPlayers 인스펙터에서 카메라 설정 > sensitivity로 회전 감도 설정이 가능합니다.

감사합니다.
